UPGRADE-GUIDE.md

```diff
@@ -0,0 +1,1298 @@
+- `--pf-c-card__header--FontWeight` to `--pf-c-card__title--FontWeight`
+- `--pf-c-card__header--not--last-child--PaddingBottom` to `--pf-c-card__title--not--last-child--PaddingBottom`
+- `--pf-c-card__header--not--last-child--PaddingBottom` to `--pf-c-card__title--not--last-child--PaddingBottom`
+
+### Chip
+- Refactor styles [(#2941)](https://github.com/patternfly/patternfly/pull/2941).
+- Removed hover, active, focus, and disabled from workspace [(#2975)](https://github.com/patternfly/patternfly/pull/2975).
+- Adjusted chip top/bottom padding, chip group margin in select [(#3079)](https://github.com/patternfly/patternfly/pull/3079).
+
+Updated markup:
+- The "x" icon in the close button changed from `fa-times-circle` to `fa-times`.
+
+Updated CSS:
+- `--pf-c-chip--BorderColor` changed from `--pf-global--secondary-color--100` to `--pf-global--BorderColor--300`
+
+Removed classes:
+- `.pf-m-read-only`
+  - Just omit the close button instead.
+- `.pf-m-hover` from `.pf-c-chip.pf-m-overflow`
+- `.pf-m-active` from `.pf-c-chip.pf-m-overflow`
+- `.pf-m-focus` from `.pf-c-chip.pf-m-overflow`
+  - The `:hover`, `:active` and `:focus` selectors still have styles applied to them.
+
+Renamed variables:
+- `--pf-c-chip--BorderColor` to `--pf-c-chip--before--BorderColor`
+- `--pf-c-chip--BorderWidth` to `--pf-c-chip--before--BorderWidth`
+- `--pf-c-chip--BorderRadius` to `--pf-c-chip--before--BorderRadius`
+- `--pf-c-chip--m-overflow--c-button--hover--BorderColor` to `--pf-c-chip--m-overflow--hover--before--BorderColor`
+- `--pf-c-chip--m-overflow--c-button--active--BorderColor` to `--pf-c-chip--m-overflow--active--before--BorderColor`
+- `--pf-c-chip--m-overflow--c-button--focus--BorderColor` to `--pf-c-chip--m-overflow--focus--before--BorderColor`
+- `--pf-c-chip--c-badge--MarginLeft` to `--pf-c-chip__c-badge--MarginLeft`
+
+Removed variables:
+- `--pf-c-chip--m-read-only--PaddingTop`
+- `--pf-c-chip--m-read-only--PaddingRight`
+- `--pf-c-chip--m-read-only--PaddingBottom`
+- `--pf-c-chip--m-overflow--BackgroundColor`
+- `--pf-c-chip--m-overflow--PaddingLeft`
+- `--pf-c-chip--m-overflow--BorderWidth`
+- `--pf-c-chip--m-overflow--c-button--BorderRadius`
+- `--pf-c-chip--m-overflow--c-button--BorderWidth`
+- `--pf-c-chip--m-overflow--c-button--PaddingLeft`
+- `--pf-c-chip--m-overflow--c-button--PaddingRight`
+- `--pf-c-chip--m-overflow--c-button--hover--BorderWidth`
+- `--pf-c-chip--m-overflow--c-button--hover--BorderColor`
+- `--pf-c-chip--m-overflow--c-button--active--BorderWidth`
+- `--pf-c-chip--m-overflow--c-button--active--BorderColor`
+- `--pf-c-chip--m-overflow--c-button--focus--BorderWidth`
+- `--pf-c-chip--m-overflow--c-button--focus--BorderColor`
+- `--pf-c-select__toggle-wrapper--c-chip--c-button--PaddingTop`
+- `--pf-c-select__toggle-wrapper--c-chip--c-button--PaddingBottom`
+
+### Chip group
+- Clarify chip group behavior [(#2961)](https://github.com/patternfly/patternfly/pull/2961)
```

Comment:
Should "Clarify" be "Clarified" ?

Reply:
I just tracked the issue back to the design docs... I've rephrased to "Clarified chip group types, overflow, and categories"